docs/designers-developers/developers/block-api/block-registration.md

```diff
@@ -486,7 +486,7 @@ transforms: {
 
 * **Type:** `Array`
 
-Blocks are able to be inserted into blocks that use [`InnerBlocks`](/packages/editor/src/components/inner-blocks/README.md) as nested content. Sometimes it is useful to restrict a block so that it is only available as a nested block. For example, you might want to allow an 'Add to Cart' block to only be available within a 'Product' block.
+Blocks are able to be inserted into blocks that use [`InnerBlocks`](/packages/block-editor/src/components/inner-blocks/README.md) as nested content. Sometimes it is useful to restrict a block so that it is only available as a nested block. For example, you might want to allow an 'Add to Cart' block to only be available within a 'Product' block.
```

Comment:
mmm, it looks like we have a bunch of links pointing to the wrong URLs. We should either:

1) Use full URLs for files that aren't published to the handbook (all component's READMEs, things that live within `*/src/*`, etc.): https://wordpress.org/gutenberg/handbook/designers-developers/developers/packages/packages-block-editor/#InnerBlocks

2) Use relative URLs to the handbook docs: `/docs/designers-developers/developers/packages/packages-block-editor/#InnerBlocks`